fala/templates/feedback/index.html

```diff
@@ -27,11 +30,3 @@ <h1 class="govuk-heading-xl">Your feedback</h1>
   </form>
 {% endblock %}
 
-{% block after_main_js %}
-  {{ super() }}
-  {% if GA_ID and error %}
-    <script>
-      ga('send', 'event', 'error', 'feedback-error');
```

Comment:
Note: This doesn't seem to be running, or maybe the whole page is not running as I couldn't find the feedback form. But I checked GA and the event wasn't pulled in anyway, so removed.